packages/synapse-react-client/src/lib/utils/theme/palette/Palettes.ts

```diff
@@ -116,10 +120,14 @@ export const adKnowledgePortalPalette: PaletteOptions = {
   secondary: generatePalette('#2f8e94'),
 }
 
+const elPortalPrimaryPalette = {
+  ...generatePalette('#39AC97'),
+  contrastText: '#ffffff',
```

Comment:
Aha, this makes sense! Is it possible to make this change (and remove the modified `contrastThreshold`) in 
- The ARK Portal palette also defined in this file
- SageAccountWeb (SourceApp.tsx) (this might not be possible today with palettes coming from the Synapse table)